src/loaddb/load_db.c

```diff
@@ -186,13 +186,27 @@ ldr_validate_object_file (const char *argv0, load_args * args)
       return ER_FAILED;
     }
 
-  if (args->cs_mode && (args->load_only == true
-			|| args->index_file.empty () == false
-			|| args->schema_file.empty () == false
-			|| args->schema_file_list.empty () == false || args->trigger_file.empty () == false))
+  if (args->cs_mode)
     {
-      fprintf (stderr, "In loaddb CS mode (-C, --CS-mode), only object loading is possible.\n");
-      return ER_FAILED;
+      if (sysprm_load_and_init (args->volume.c_str (), NULL, SYSPRM_IGNORE_INTL_PARAMS) == NO_ERROR)
+	{
+	  if (prm_get_integer_value (PRM_ID_HA_MODE))
```

Comment:
what about changing this expression?
I knew it is referenced from legacy code. But I remember that we usually use expression in the if statement.
```suggestion
          if  (args->cs_mode == true)
	  if (prm_get_integer_value (PRM_ID_HA_MODE) == true)
```

Reply:
And what about use in one if statement below two statements?
Could you tell me why you choose this way?
```suggestion
	  if ((prm_get_integer_value (PRM_ID_HA_MODE)==true) &&  (args->load_only == true)
	  ||
```

Reply:
The two doors have different meanings, so they are separated.

Reply:
> The two doors have different meanings, so they are separated.
I see.
in case 'prm_get_integer_value (PRM_ID_HA_MODE)' returns nothing but args->load_only == true returns something.

I can guess the case of 'prm_get_integer_value (PRM_ID_HA_MODE) == false'  not the return case.

Reply:
The prm_get_integer_value () function returns TREU or FALSE. I made it so that only OBJECT can be loaded when it is TRUE. If FALSE, there is no need to check the data type. so it was separated
The reason is to prevent schema loading when in HA Mode. See "Purpose" above for details.